core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/solver/change/DefaultProblemChangeDirector.java

```diff
@@ -108,9 +107,10 @@ public DefaultProblemChangeDirector(InnerScoreDirector<Solution_, ?> scoreDirect
         return Optional.ofNullable(scoreDirector.lookUpWorkingObjectOrReturnNull(externalObject));
     }
 
-    public Score<?> doProblemChange(ProblemChange<Solution_> problemChange) {
+    public void doProblemChange(ProblemChange<Solution_> problemChange, boolean triggerVariableListeners) {
```

Comment:
I'm not sure why we have the boolean here, if we already have the information on `ProblemChange`.